### ISSUE TITLE
VSCode Plugin: Multiple Configuration Files

### ISSUE THREAD

**Issue by alexlafroscia** [apollographql/apollo-tooling#802]
I have a monorepo with a few different GraphQL projects in it. Because of this, I need to configure different parts of the project to use different local `schema.graphql` files.

[`graphql-config`](https://github.com/prisma/graphql-config) seems to support this use-case nicely, but unfortunately the VSCode plugin does not actually provide auto-complete and the like, which I'm looking for (or at least, this is broken right now). So I'm trying to see if I can get the Apollo plugin installed instead.

I have an `apollo.config.js` file within one of the projects in my monorepo, and the file is discovered at least. Unfortunately, all of the relative paths are not relative to the config file, but rather the project root. So, the file looks like this:

```javascript
module.exports = {
  client: {
    includes: [
      './packages/original-series-engine/addon/**/*.graphql',
      './packages/original-series-engine/addon/**/*.js'
    ],
    service: {
      name: 'Original Series Workflow API',
      localSchemaFile:
        'packages/original-series-engine/addon/graphql/schema.graphql'
    }
  }
};
```

Even though the file itself lives at `packages/original-series-engine/apollo.config.js`.

Is there a different way to configure different schemas for different parts of the project, so I can move the config file up to the root of the project? If not, is there a way to "nicely" support multiple files, with relative paths that make sense?

**Comment by JakeDawkins** [comment]
A couple things here :) 

Right now, we don't support a list of projects in the config file, so each package in a repo should have its own config file.

Also, relative globs have been broken for a while in the `includes`/`excludes` fields. They should be fixed when #1007 is released. The issue of the globs being relative to the root _shouldn't_ be an issue anymore, but I can reopen this issue if it is still a problem

**Comment by djgrant** [comment]
I bumped into this issue and was able to resolve it by providing an absolute path. As the config is in a JS module, you can resolve the relative path using `path.resolve`:

```js
const path = require('path');

module.exports = {
  client: {
    service: {
      name: 'my-service',
      localSchemaFile: path.resolve(
        __dirname,
        '../path/to/my/schema.graphql',
      ),
    },
  },
};
```
